frontend/views/utils/translations.js

```diff
@@ -65,6 +66,20 @@ export default function L (
   return template(key, args)
 }
 
+export function LError (error) {
+  if (error instanceof GIErrorUIRuntimeError) {
+    // It's a controlled/predictable error, so we don't need the user to report it to us.
+    return { reportError: error.message }
+  }
+  return {
+    reportError: L('"{errorMsg}". {a_}Report the error{_a}.', {
```

Comment:
If you want, I recommend additionally adding a key for `errorMsg` at the top level here too (along with the previous `troubleshootingLink`, so that `LError` usage can be made more flexible if you want it.

Reply:
I didn't understand what you mean nor the flexibility advantage... 😕

Reply:
So sometimes you want this exact string in `reportError` that you have here, but other times you might want to write a different string, e.g. have the positioning of `errorMsg` be different.

Reply:
So sometimes you want this exact string in `reportError` that you have here, but other times you might want to write a different string, e.g. have the positioning of `errorMsg` be different.

Reply:
> but other times you might want to write a different string.

Do we? Where? Maybe I missed it. Anyway, I thought we agreed to do it as it is, to keep consistency and DRY. 

Quote from https://github.com/okTurtles/group-income-simple/pull/881#discussion_r405861003:
> What if error, comes from {reportError}? That way we don’t need to repeat the 2 variables everywhere.